.idea/copyright/AB.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="Copyright (c) &amp;#36;{today.year} AccelByte Inc. All Rights Reserved.&#10;This is licensed software from AccelByte Inc, for limitations&#10;and restrictions contact your company contract manager." />
```

Comment:
use the appropriate template to accommodate Apache license header

```
// Copyright [yyyy] [name of copyright owner]
// 
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
// 
//     http://www.apache.org/licenses/LICENSE-2.0
// 
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
and don't forget to update all files copyright header, I see all of them are using an exclusive copyrighted header

Reply:
@bahrunnur do we already use proposed headers for other public repositories?

Reply:
I haven't checked it all, but some of it are using Apache headers in their individual files for example:

1. https://github.com/AccelByte/public-source-ip/blob/master/public-source-ip.go
2. https://github.com/AccelByte/augment-python-sdk
3. https://github.com/AccelByte/iam-go-sdk

And this one is not using Apache header, even though the LICENSE itself is Apache: https://github.com/AccelByte/profanity-filter-go

Reply:
@bahrunnur let's apply it then :+1: